packages/controls/src/controls/group/__snapshots__/group.test.ts.snap

```diff
@@ -201,40 +208,19 @@ exports[`ShapeV2 serialization serialize/deserialize composable shape of control
             },
           },
         },
-        "type": "makeswift::controls::shape-v2",
+        "type": "makeswift::controls::group",
       },
     },
   },
-  "type": "makeswift::controls::shape-v2",
-}
-`;
-
-exports[`ShapeV2 serialization serialize/deserialize label of controls 1`] = `
```

Comment:
For some reason Jest decided to change the order of these snapshots, they are [here](https://github.com/makeswift/makeswift/pull/909/files?diff=split&w=1#diff-2705afb1a9c0c520d1fa6248c04dc6dc4c45cde9c10263baf6cc38f71f3790c7R242).